README.md

```diff
@@ -311,14 +324,24 @@ This will result in Kafka Broker making a request to authorization server every
 Specify the following `oauth.*` properties:
 - `oauth.introspection.endpoint.uri` (e.g.: "https://localhost:8443/auth/realms/demo/protocol/openid-connect/token/introspect")
 - `oauth.valid.issuer.uri` (e.g.: "https://localhost:8443/auth/realms/demo" - only access tokens issued by this issuer will be accepted)
-- `oauth.client.id` (e.g.: "kafka" - this is the client configuration id for Kafka Broker)
+- `oauth.client.id` (e.g.: "kafka" - this is the OAuth2 client configuration id for the Kafka Broker)
 - `oauth.client.secret` (e.g.: "kafka-secret")
  
 Introspection endpoint should be protected. The `oauth.client.id` and `oauth.client.secret` specify Kafka Broker credentials for authenticating to access the introspection endpoint. 
 
 Some authorization servers don't provide the `iss` claim. In that case you would not set `oauth.valid.issuer.uri`, and you would explicitly turn off issuer checking by setting the following option to `false`:
 - `oauth.check.issuer` (e.g.: "false")
 
+You can enforce audience checking, which is an OAuth2 mechanism to prevent successful authentication with tokens that are explicitly issued for use by your resource server.
```

Comment:
```suggestion
You can enforce audience checking, which is an OAuth2 mechanism to prevent successful authentication with tokens that are not explicitly issued for use by your resource server.
```

Reply:
Again ...
```suggestion
You can enforce audience checking, which is an OAuth2 mechanism to prevent successful authentication unless tokens are explicitly issued for use by your resource server.
```